decidim-templates/app/forms/decidim/templates/admin/proposal_answer_template_form.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Templates
+    module Admin
+      class ProposalAnswerTemplateForm < TemplateForm
+        attribute :internal_state, String
+        attribute :scope_for_availability, String
+
+        validates :internal_state, presence: true
+
+        def map_model(model)
+          self.scope_for_availability = "#{model.templatable_type.try(:demodulize).try(:tableize)}-#{model.templatable_id.to_i}"
```

Comment:
It seems like we're hacking a String a bit... Maybe we can use a `JsonbAttribute` type here? Or do you feel that's overkill for the moment?

Reply:
I have changed this, and we are not hacking it anymore